cmd/service-account-issuer-discovery/main.go

```diff
@@ -15,12 +16,14 @@ import (
 	"time"
 
 	"github.com/gardener/service-account-issuer-discovery/internal/app"
+	"github.com/gardener/service-account-issuer-discovery/internal/version"
 
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 )
 
 var (
+	buildInfo            = flag.Bool("version", false, "Build information for the currently running binary")
```

Comment:
I think that `showVersion` is a better name for this variable.

```suggestion
	showVersion          = flag.Bool("version", false, "Print the server version information.")
```